handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -129,7 +130,7 @@ protected final void onLookupComplete(ChannelHandlerContext ctx,
     protected void replaceHandler(ChannelHandlerContext ctx, String hostname, SslContext sslContext) throws Exception {
         SslHandler sslHandler = null;
         try {
-            sslHandler = sslContext.newHandler(ctx.alloc());
+            sslHandler = newSslHandler(sslContext, ctx.alloc());
```

Comment:
shouldn't we be passing hostname through to the ssl handler?

Reply:
@johnou I dont think so as the hostname overload is only used for client-side. (also we did not before as well).